README.md

```diff
@@ -17,35 +17,11 @@ Code](https://img.shields.io/tokei/lines/github/osmosis-labs/osmosis?style=flat-
-can offer decentralized formation of token fundraisers, interchain
-staking, options market, and more for the Cosmos ecosystem.
-
-Whereas most Cosmos zones have focused their incentive scheme on the
-delegators, Osmosis attempts to align the interests of multiple
-stakeholders of the ecosystem such as LPs, DAO members, as well as
-delegators. One mechanism that is introduced is how staked liquidity
-providers have sovereign ownership over their pools, and through the
-pool governance process allow them to adjust the parameters depending on
-the pool's competition and market conditions. Osmosis is a sovereign
-Cosmos zone that derives its sovereignty not only from its
-application-specific blockchain architecture but also the collective
-sovereignty of the LPs that has aligned interest to different tokens
-that they are providing liquidity for.
+As the largest DEX in the Cosmos, Osmosis is a source of liquidity for over 50 sovereign blockchains connected via IBC. Pioneering in its approach, Osmosis offers a dynamic trading and liquidity provision experience, integrating non-IBC assets from other ecosystems, including Ethereum, Solana, Avalanche, and Polkadot. Initially adopting Balancer-style pools, Osmosis now also features a concentrated liquidity model that is orders of magnitude more capital efficient, meaning that significantly less liquidity is required to handle the same amount of trading volume with minimal slippage.
+
+As a true Layer 1 appchain, Osmosis has greater control over the full blockchain stack than traditional smart contract DEXs, which must follow the code of the parent chain that it is built on. This fine-grained control has enabled, for example, the development of Superfluid Staking, an extension of Proof of Stake that allows assets at the application layer to be staked to secure the chain. The customizability of appchains also allows implementing features like the Protocol Revenue module, which enables Osmosis to conduct on-chain arbitrage on behalf of OSMO stakers, balancing prices across pools while generating real yield revenue from this volume. Additionally, as a sovereign appchain, Osmosis governance can vote on upgrades to the protocol. One example of this was the introduction of a Taker Fee, which switched on the collection of exchange fees to generate diverse yield from Osmosis volume and distribute it to OSMO stakers.
+
+Osmosis is bringing the full centralized exchange experience to the decentralized world by building a cross-chain native DEX and trading suite that connects all chains over IBC, including Ethereum and Bitcoin. To reach this goal, Osmosis hosts an ever-expanding suite of DeFi applications aimed at providing a one-stop experience that includes lending, credit, margin, DeFi strategy vaults, power perps, fiat on-ramps, NFTs, stablecoins, and more — all of the functionalities that centralized exchange offer and more, in the trust-minimized environment of decentralized finance.
```

Comment:
> all chains over IBC, including Ethereum and Bitcoin

This feels a bit suspicious as Ethereum and Bitcoin aren't connected via IBC precisely

Reply:
Sorry about this. I didn't write the original but I think what I've rewritten should be more accurate now